custom_components/isy994/__init__.py

```diff
@@ -81,8 +81,7 @@
     SUPPORTED_VARIABLE_PLATFORMS,
     UNDO_UPDATE_LISTENER,
 )
-
-_LOGGER = logging.getLogger(__name__)
```

Comment:
I think going to make the logger log everything as `isy994.const` instead

Reply:
Const has `__package__` so it's logging as `custom_components.isy994`